sdk/go/pulumi/generate/templates/types_builtins_test.go.template

```diff
@@ -317,56 +317,56 @@ func TestBuiltinConversions(t *testing.T) {
 	assetOrArchiveOut := archiveIn.ToAssetOrArchiveOutput()
 	archiveV, known, _, _, err := await(assetOrArchiveOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, archiveIn, archiveV)
 
 	archiveOut := archiveIn.ToArchiveOutput()
 	assetOrArchiveOut = archiveOut.ToAssetOrArchiveOutput()
 	archiveV, known, _, _, err = await(assetOrArchiveOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, archiveIn, archiveV)
 
 	assetIn := NewFileAsset("foo.zip")
 	assetOrArchiveOut = assetIn.ToAssetOrArchiveOutput()
 	assetV, known, _, _, err := await(assetOrArchiveOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, assetIn, assetV)
 
 	assetOut := assetIn.ToAssetOutput()
 	assetOrArchiveOut = assetOut.ToAssetOrArchiveOutput()
 	assetV, known, _, _, err = await(assetOrArchiveOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, assetIn, assetV)
 
 	idIn := ID("foo")
 	stringOut := idIn.ToStringOutput()
 	stringV, known, _, _, err := await(stringOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, string(idIn), stringV)
 
 	idOut := idIn.ToIDOutput()
 	stringOut = idOut.ToStringOutput()
 	stringV, known, _, _, err = await(stringOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, string(idIn), stringV)
 
 	urnIn := URN("foo")
 	stringOut = urnIn.ToStringOutput()
 	stringV, known, _, _, err = await(stringOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, string(urnIn), stringV)
 
 	urnOut := urnIn.ToURNOutput()
 	stringOut = urnOut.ToStringOutput()
 	stringV, known, _, _, err = await(stringOut)
 	assert.True(t, known)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, string(urnIn), stringV)
```

Comment:
The ordering of these ones is a bit odd; not sure what we are generating but appreciate this is just a mechanical search/replace.